portfolio/src/main/webapp/script.js

```diff
@@ -48,6 +48,40 @@ function makeCollapsible() {
+  var markerFlorida =   new google.maps.Marker({position: {lat: 28.5383, lng: -81.3792}, map: map, icon: greenPin});
+  var markerTexas =     new google.maps.Marker({position: {lat: 29.4241, lng: -98.4936}, map: map, icon: greenPin});
+  var markerLA =        new google.maps.Marker({position: {lat: 34.0522, lng: -118.2437}, map: map, icon: greenPin});
+  var markerSanFran =   new google.maps.Marker({position: {lat: 37.7749, lng: -122.4194}, map: map, icon: greenPin});
+  var markerPitt =      new google.maps.Marker({position: {lat: 40.4406, lng: -79.9959}, map: map, icon: greenPin});
+  var markerYellow =    new google.maps.Marker({position: {lat: 44.4280, lng: -110.5885}, map: map, icon: greenPin});
+  var markerAriz =      new google.maps.Marker({position: {lat: 36.0544, lng: -112.1401}, map: map, icon: greenPin});
+  var markerDenver =    new google.maps.Marker({position: {lat: 39.7392, lng: -104.9903}, map: map, icon: greenPin});
+  var markerToronto =   new google.maps.Marker({position: {lat: 43.6532, lng: -79.3832}, map: map, icon: greenPin});
+  var markerSask =      new google.maps.Marker({position: {lat: 52.1332, lng: -106.6700}, map: map, icon: greenPin});
+  var markerBanff =     new google.maps.Marker({position: {lat: 51.1784, lng: -115.5708}, map: map, icon: greenPin});
+  var markerJeru =      new google.maps.Marker({position: {lat: 31.7683, lng: 35.2137}, map: map, icon: greenPin});
+  var markerSafed =     new google.maps.Marker({position: {lat: 32.9646, lng: 35.4960}, map: map, icon: greenPin});
+  var markerDnipro =    new google.maps.Marker({position: {lat: 48.4647, lng: 35.0462}, map: map, icon: greenPin});
+  var markerKiev =      new google.maps.Marker({position: {lat: 50.4501, lng: 30.5234}, map: map, icon: greenPin});
+  var markerIceland =   new google.maps.Marker({position: {lat: 64.9631, lng: -19.0208}, map: map, icon: purplePin});
+  var markerNewZ =      new google.maps.Marker({position: {lat: -40.9006, lng: 174.8860}, map: map, icon: purplePin});
+  var markerSAfrica =   new google.maps.Marker({position: {lat: -30.5595, lng: 22.9375}, map: map, icon: purplePin});
+  var markerAustralia = new google.maps.Marker({position: {lat: -25.2744, lng: 133.7751}, map: map, icon: purplePin});
+  var markerGreece =    new google.maps.Marker({position: {lat: 39.0742, lng: 21.8243}, map: map, icon: purplePin});
+  var markerItaly =     new google.maps.Marker({position: {lat: 41.8719, lng: 12.5674}, map: map, icon: purplePin});
+  var markerParis =     new google.maps.Marker({position: {lat: 48.8566, lng: 2.3522}, map: map, icon: purplePin});
+  var markerLondon =    new google.maps.Marker({position: {lat: 51.5074, lng: -0.1278}, map: map, icon: purplePin});  
+  var markerHawaii =    new google.maps.Marker({position: {lat: 19.8968, lng: -155.5828}, map: map, icon: purplePin});  
+  var markerTokyo =     new google.maps.Marker({position: {lat: 35.6762, lng: 139.6503}, map: map, icon: purplePin});  
```

Comment:
Anything we can use to store multiples of something?

Reply:
One other idea, generally you'd want data items like these to be stored in an easily maintainable format that's separate from business logic operating on them.  Database, csv, or json are some options.  If you want to keep these on the client side, maybe store them in a separate *.js file and import the values here.  If you want to store these on the server, you can have the servlet read the data file and send it back as part of the response.